plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintExtension.kt

```diff
@@ -47,5 +54,7 @@ open class KtlintExtension {
      *
      * Default is plain.
      */
-    var reporters: Array<ReporterType> = arrayOf(ReporterType.PLAIN)
+    val reporters: SetProperty<ReporterType> = objectFactory.setProperty {
```

Comment:
@Jlleitschuh Changing this to `SetProperty<ReporterType> bring problem that in common gradle script it is not allowed anymore to use `reporters = ["CHECKSTYLE", "PLAIN"]` and I don't know for now how address it better.

See example how it looks now here: https://github.com/JLLeitschuh/ktlint-gradle/blob/094995594f19bf25a2f4d94797afb5f72a09b317/samples/kotlin-gradle/build.gradle#L16

Reply:
Possible solution is to use `SetProperty<String>` in extention and convert it to `SetProperty<ReporterType>` when passing to the task.

On the other hand, now, it is more obvious what reporter types are available, and has the same approach as kotlin dsl extention configuration.

@JLLeitschuh what do you think about it?